src/Error.js

```diff
@@ -1,13 +1,13 @@
-'use strict'
-
 class SyntaxError extends Error {
   constructor (err) {
     super(err)
 
-    this.name = 'Syntax Error'
+    const { line, column, reason } = err
+
+    this.name = 'SyntaxError'
 
     this.message = ''
-    this.message += `${this.name} \n\n(${err.line}:${err.column}) ${err.reason}`
+    this.message += `${this.name} \n\n(${line}:${column}) ${reason}`
```

Comment:
You should use `err.reason ? err.reason : err.message` instead `SyntaxName`, not all errors is syntax.

You should check on `line/column` here, because not all error contain `line` and `column`

Reply:
I check for it within the loader already via `err.name = 'CssSyntaxError' ? new SyntaxError(err) : err` so the properties are always defined